packages/feedback/src/modal/components/DialogHeader.tsx

```diff
@@ -6,17 +6,16 @@ import { useMemo } from 'preact/hooks';
 import { SentryLogo } from './SentryLogo';
 
 export interface Props {
-  formTitle: FeedbackInternalOptions['formTitle'];
-  showBranding: FeedbackInternalOptions['showBranding'];
+  options: FeedbackInternalOptions;
 }
 
-export function DialogHeader({ formTitle, showBranding }: Props): VNode {
```

Comment:
l: i kind of like having the specific props, esp for this component where it only needs 2 of the options